### PR TITLE
Add Inventory Frontend

### PR DESCRIPTION
**Pull request by aashrav**
Created inventory_ui
- Doesn't connect to database yet(all items are hard-coded for now)
- Search bar and category filtering are working
- Still need to add: edit item & add item capabilities 

image

### CODE REVIEW

File: src/Pages/InventoryItem/AddEditModal.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+
+const AddEditModal = (props) => {
+  return (
+    <Modal isOpen={props.open} toggle={props.toggle}>
+      <ModalHeader toggle={props.toggle}>{props.title}</ModalHeader>
+      <ModalBody className="add-item-body">
+        <form>
+          <label>Name:</label>
```
Comment (evanugarte) [inline src/Pages/InventoryItem/AddEditModal.js:10]: homie i know you can use Array.map [like this](https://github.com/SCE-Development/Core-v4/blob/addd3e75b606aeefca6cfef47d7d4d56dc6810b8/src/Pages/LedSign/LedSign.js#L17)

File: src/Pages/InventoryItem/AddEditModal.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+          {console.log(props.data)}
```
Comment (evanugarte) [inline src/Pages/InventoryItem/AddEditModal.js:11]: remove

File: src/Pages/InventoryItem/EditItemModal.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+          <label>Category:</label>
+          <input id="catInput"   defaultValue = {props.data.cat} type="text" required></input>
+
+          <label>Picture URL:</label>
+          <input id="picInput" defaultValue = {props.data.pic} type="url"></input>
+        </form>
+      </ModalBody>
+      <ModalFooter>
+        {/* <button
```
Comment (evanugarte) [inline src/Pages/InventoryItem/EditItemModal.js:31]: remove comment if you arent using this

File: src/Pages/InventoryItem/EditItemModal.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+import {Modal, ModalHeader, ModalBody, ModalFooter} from 'reactstrap';
+
+const EditItemModal = (props) => {
+  console.log(props.open)
+  return (
+    <Modal isOpen={props.open} toggle={props.toggle}>
+      <ModalHeader toggle={props.toggle}>Edit Item</ModalHeader>
+      <ModalBody className="add-item-body">
+        <form>
+          <label>Name:</label>
```
Comment (evanugarte) [inline src/Pages/InventoryItem/EditItemModal.js:11]: array.map here please

File: src/Pages/InventoryItem/EditItemModal.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {Modal, ModalHeader, ModalBody, ModalFooter} from 'reactstrap';
+
+const EditItemModal = (props) => {
+  console.log(props.open)
```
Comment (evanugarte) [inline src/Pages/InventoryItem/EditItemModal.js:5]: remove

File: src/Pages/InventoryItem/InventoryHeading.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+];
+
+const InventoryHeading = (props) => {
+  return (
+    <tr className = "inventory-heading">
+      {fields.map((ele, index) => {
+        if (index === 4) {
+          return(<td key={index}>
+            <form >
+              {'Category: '}
```
Comment (evanugarte) [inline src/Pages/InventoryItem/InventoryHeading.js:20]: should this be in an `h1` or other tag?

File: src/Pages/InventoryItem/InventoryHeading.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+  'Stock',
+  'Category',
+  'Picture',
+  'Edit'
+];
+
+const InventoryHeading = (props) => {
+  return (
+    <tr className = "inventory-heading">
+      {fields.map((ele, index) => {
```
Comment (evanugarte) [inline src/Pages/InventoryItem/InventoryHeading.js:16]:
```suggestion
      {fields.map((fieldName, index) => {
```

File: src/Pages/InventoryItem/InventoryTable.js [hunk 8]
```diff
@@ -0,0 +1,164 @@
+import React, { Component } from 'react';
+import './inventory-table.css';
+import ItemList from './ItemList';
+import InventoryHeading from './InventoryHeading.js';
+// import {Modal, ModalHeader, ModalBody, ModalFooter, Label} from 'reactstrap'
```
Comment (evanugarte) [inline src/Pages/InventoryItem/InventoryTable.js:5]: remove this and the line below if you arent gonna use it

**Comment by ThaiQ** [comment]
@aashrav start to integrate your backend. also rebase to the latest commit to fix Travis' grpc

**Pull request closed without merging**